### PR TITLE
IDBA-UD-1.1.1-goolf-1.4.10.eb

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**SUCCESS**
Build succeeded for 1 out of 1
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/839c2068a58c1818a496 for a full test report.

File: easybuild/easyconfigs/i/IDBA-UD/IDBA-UD-1.1.1-goolf-1.4.10.eb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+files_to_copy = ["bin", "script", "README", "ChangeLog", "NEWS"]
+        
+sanity_check_paths = {
+    'files': ["bin/%s" % x for x in ["idba", "idba_hybrid", "idba_tran",
+                                    "idba_ud", "parallel_blat", "idba_tran_test"]],
+    'dirs': [""],
+}
+
+moduleclass = 'bio'
+
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/i/IDBA-UD/IDBA-UD-1.1.1-goolf-1.4.10.eb]: nice and elegant, just trim last empty line and prefer `'dirs': [],`

**Comment by boegel** [comment]
Jenkins: test this please

File: easybuild/easyconfigs/i/IDBA-UD/IDBA-UD-1.1.1-goolf-1.4.10.eb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+ locally to generate some missing k-mers in low-depth regions. With these technologies, IDBA-UD 
+ can iterate k value of de Bruijn graph to a very large value with less gaps and less branches 
+ to form long contigs in both low-depth and high-depth regions."""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['http://hku-idba.googlecode.com/files/']
+sources = ['idba-%(version)s.tar.gz']
+
+premakeopts = './configure && '
```
Comment (boegel) [inline easybuild/easyconfigs/i/IDBA-UD/IDBA-UD-1.1.1-goolf-1.4.10.eb]: prebuildopts

File: easybuild/easyconfigs/i/IDBA-UD/IDBA-UD-1.1.1-goolf-1.4.10.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/i/IDBA-UD/IDBA-UD-1.1.1-goolf-1.4.10.eb]: +1

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/80f61d30d1a2ef27d929 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @pescobar!

**Pull request merged**